Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ElasticWindowMultiple.py

```diff
@@ -210,7 +208,7 @@ def axis_value(index):
                 elt_workspace = CloneWorkspace(InputWorkspace=elf_workspace, OutputWorkspace="__cloned",
                                                StoreInADS=False, EnableLogging=False)
 
-            _normalize_by_index(elt_workspace, np.argmin(sample_param))
+            _normalize_by_index(elt_workspace, 0)
```

Comment:
Is there a reason that this doesn't require a test to be updated.

Reply:
Currently there are no tests for the ElasticWindowMultiple algorithm,

Reply:
Ok. Could you make an issue for that?